index.d.ts

```diff
@@ -498,14 +498,92 @@ declare namespace dashjs {
     }
 
     export interface Period {
-        id : string | null;
+        id: string | null;
         index: number;
         duration: number;
         start: number;
         mpd: Mpd;
         nextPeriodId: string | null;
     }
 
+    export interface IAdaption {
+        ContentProtection: IContentProtection[];
+        ContentProtection_asArray: IContentProtection[] // Why? ContentProtection is already type array
```

Comment:
Depends on how many entires there are. If there is only one entry `ContentProtection` is an object not an array. This is one of the reasons we are replacing the XML parser in v.5

Reply:
ok understood. Was just wondering the duplication I have always seen :)